kolibri/core/assets/src/core-actions.js

```diff
@@ -170,6 +246,24 @@ function initContentSession(store, Kolibri, channelId, contentId, contentKind) {
       /* If a summary model exists, map that to the state */
       if (summary.length > 0) {
         store.dispatch('SET_LOGGING_SUMMARY_STATE', _contentSummaryLoggingState(summary[0]));
+        if (summary[0].currentmasterylog) {
```

Comment:
(there's actually only ever going to be either 0 or 1, in this case -- but we retrieve it via a filter on the full queryset, so we get a list, and need to index in)